pkg/schema/tracker.go

```diff
@@ -70,6 +70,8 @@ func NewTracker(ctx context.Context, task string, sessionCfg map[string]string,
 	// bypass wait time of https://github.com/pingcap/tidb/pull/20550
 	toSet.TiKVClient.AsyncCommit.SafeWindow = 0
 	toSet.TiKVClient.AsyncCommit.AllowedClockDrift = 0
+	// set a negative lease will disable the background statistic jobs.
```

Comment:
Maybe add a link to tidb to explain how this change is work

https://github.com/pingcap/tidb/blob/7755d25aba5120dafde98fff11ab3b98ca4d192f/domain/domain.go#L1176

Reply:
or we can upgrade `tidb` version to master, it provides a func `SetStatsLease` which can help us do this config

https://github.com/pingcap/tidb/blob/7755d25aba5120dafde98fff11ab3b98ca4d192f/session/tidb.go#L156

Reply:
The code are changing so we'll give an inactive reference soon.

And I don't the the right time to call `SetStatsLease`

Reply:
changed to function call in https://github.com/pingcap/dm/pull/2065/commits/7ed00726787fe527e07930cb27c4fdde0b0c1771